Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -6,14 +6,18 @@
 #
 # Description:
 #    This script was created to automate the testing of a Linux
-#    Integration services. This script will verify if all the CPUs 
-#    inside a Linux VM are processing VMBus interrupts, by checking 
+#    Integration services. This script will verify if all the CPUs
+#    inside a Linux VM are processing VMBus interrupts, by checking
 #    the /proc/interrupts file.
 #
 #    The test performs the following steps:
 #        1. Looks for the Hyper-v timer property of each CPU under /proc/interrupts
 #        2. Verifies if each CPU has more than 0 interrupts processed.
-#     
+#           a) Hypervisor callback interrupts: vmbus messages/events. It could have CPU
+#              with 0 interrupts in big VM size.
+#           b) Hyper-V reenlightenment interrupts: Only used in Nexted virtualization.
+#           c) Hyper-V stimer0 interrupts: new Hyper-V timer, and if exsts, this count
```

Comment:
exsts -> exists

Reply:
Fixed in 592ddddb5cad9550a23dd4f9b39ffb836f03fb41